### ISSUE TITLE
Sideloading

### ISSUE THREAD

**Issue by brayniverse** [rtablada/eloquent-ember#1]
My understand is that when sideloading objects ember expects a JSON response like the example below with the related objects outside of the root object.

```
{
  "post":
    {
      "id": "2",
      "title": "Blog Post Title",
      "created_at": "2013-08-28 16:21:44",
      "updated_at": "2013-08-28 16:21:44",
      "comment_ids": ["10"]
    }
  "comments": [
    {
      "id": "10",
      "post_id": "2",
      "name": "John Doe",
      "message": "Demonstrating stuff..."
    }
  ]
}
```

At the moment eloquent-ember is loading the comments object inside of the post. Like so:

```
{
  "post":
    {
      "id": "2",
      "title": "Blog Post Title",
      "created_at": "2013-08-28 16:21:44",
      "updated_at": "2013-08-28 16:21:44",
      "comment_ids": ["10"]
      "comments": [
        {
          "id": "10",
          "post_id": "2",
          "name": "John Doe",
          "message": "Demonstrating stuff..."
        }
      ]
    }
}
```

**Comment by rtablada** [comment]
Currently, I am not handling side loading cases instead looking at the withIds property.

I am going to work on sideloading soon.

If you can file a pull request, I can test it and merge it.

**Comment by brayniverse** [comment]
Ah okay I didn't realise, I posted it because you were loading the relationships in the response and so I assumed you hadn't noticed. I can possibly write something, I'll get back to you when it's done.

**Comment by rtablada** [comment]
Fixed by 67434b9d4875a7e97e01d76ea7e41b6046fb6fee
